Kernel/Memory/Region.cpp

```diff
@@ -604,10 +604,10 @@ PageFaultResponse Region::handle_inode_fault(size_t page_index_in_region, bool m
     if (current_thread)
         current_thread->did_inode_fault();
 
-    u8 page_buffer[PAGE_SIZE];
+    auto page_buffer = Array<u8, PAGE_SIZE> {};
```

Comment:
`Array<u8, PAGE_SIZE> page_buffer {};

Reply:
…but inode faulting probably happens fairly frequently – maybe the LHS approach of only zeroing what was missing is preferrable? (Fine to use an Array, just remove the `{}`)

Reply:
I really don't want to do this, because a couple of lines afterwards there's this:
```c++
// If we read less than a page, zero out the rest to avoid leaking uninitialized data.
    if (nread < PAGE_SIZE)
        memset(page_buffer + nread, 0, PAGE_SIZE - nread);
```

This is ugly, and in my opinion, doing this is not justified anyway if you need to zero out the buffer - just zero out it all and don't worry about it later.